python/pyspark/pandas/typedef/typehints.py

```diff
@@ -585,17 +580,12 @@ def infer_return_type(f: Callable) -> Union[SeriesType, DataFrameType, ScalarTyp
         return SeriesType(dtype, spark_type)
 
     # Note that, DataFrame type hints will create a Tuple.
-    # Python 3.6 has `__name__`. Python 3.7 and 3.8 have `_name`.
```

Comment:
There's similar instance at `python/pyspark/sql/pandas/typehints.py` too.

Reply:
Just to be sure ‒ is this what you meant 33f1901?